packages/good-design/src/core/modals/FirstClaimModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { noop } from "lodash";
+import React, { FC, useCallback, useState } from "react";
+import { Modal, Spinner, Text, View, Pressable, IModalProps } from "native-base";
+import { useFVLink } from "@gooddollar/web3sdk-v2";
+
+import CrossIcon from "./cross";
+import { openLink } from "../utils/linking";
+import ButtonAction from "../buttons/ActionButton";
+import { FVModalProps } from "../buttons/ClaimButton";
+import { withTheme, withThemingTools } from "../../theme";
+
+export interface FirstClaimModalProps extends IModalProps {}
+
+const FirstClaimModal: FC<FirstClaimModalProps> = ({ onClose = noop, ...props }) => {
+  return (
```

Comment:
if no additional logic (callbacks, hooks etc) - do not use return

e.g. const SimpleComponent = props => (
  jsx tree here
)